src/containers/AstroToolContainer.jsx

```diff
@@ -54,8 +51,8 @@ class AstroToolContainer extends React.PureComponent {
     const { data, options, selectedFilters } = this.state;
     const { filters, colorOptions, hexColors, galaxies, galaxyImgs } =
       data || {};
-    const { galaxyImg } = options || {};
-    console.log(selectedFilters);
+    const { galaxyImg, saveAnswer } = options || {};
```

Comment:
I'm thinking that instead of using `saveAnswer` as your flag, you can similarly us `questionId` and have that be your flag to trigger in the Color Tool whether or not to save the data.